airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -423,7 +424,8 @@ public boolean isWorkflowRunning(final String workflowName) {
       final ListOpenWorkflowExecutionsResponse listOpenWorkflowExecutionsRequest =
           service.blockingStub().listOpenWorkflowExecutions(openWorkflowExecutionsRequest);
       final List<WorkflowExecutionInfo> workflowExecutionInfos = listOpenWorkflowExecutionsRequest.getExecutionsList().stream()
-          .filter((workflowExecutionInfo -> workflowExecutionInfo.getExecution().getWorkflowId().equals(workflowName)))
+          .filter((workflowExecutionInfo -> workflowExecutionInfo.getExecution().getWorkflowId().equals(workflowName)
+              && workflowExecutionInfo.getStatus() == WorkflowExecutionStatus.WORKFLOW_EXECUTION_STATUS_RUNNING))
```

Comment:
What's the implication of this change? On the dev-2 instance manual runs were able to be picked up because it identified a workflow as not running.